packages/terra-functional-testing/src/commands/expect/toMatchReference.js

```diff
@@ -3,10 +3,11 @@
  * the mismatch tolerance and are the same size.
  *
  * @param {Object} screenshot - The comparison result.
- * @param {boolean} screenshot.isNewScreenshot - If the latest screenshot was a new screenshot that has no reference screenshot.
- * @param {boolean} screenshot.isSameDimensions - If the latest screenshot was the same size as the reference screenshot.
- * @param {boolean} screenshot.isWithinMismatchTolerance - If the latest screenshot was within the mismatch tolerance.
+ * @param {Boolean} screenshot.isNewScreenshot - If the latest screenshot was a new screenshot that has no reference screenshot.
+ * @param {Boolean} screenshot.isSameDimensions - If the latest screenshot was the same size as the reference screenshot.
+ * @param {Boolean} screenshot.isWithinMismatchTolerance - If the latest screenshot was within the mismatch tolerance.
  * @param {Number} screenshot.misMatchPercentage - The mismatch percentage when comparing the latest screenshot to the reference screenshot.
+ * @param {Boolean} screenshot.screenshotWasUpdated - If the reference screenshot was updated with the latest captured screenshot.
```

Comment:
nit: How do you feel about lowercasing primitives? I've been following the examples on the JSDoc page which lowercase primitive types and capitalize object types:

https://jsdoc.app/tags-param.html

Reply:
I updated these because other places are using the uppercase but looks like I followed the bad examples. Updated all the primitive types to use lowercase: https://github.com/cerner/terra-toolkit/pull/568/commits/07340103010232d3e8f52accdb9e9f65c44e0371